packer/scripts/vagrant.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Set up sudo
+echo '%vagrant ALL=(ALL) NOPASSWD:ALL' > /etc/sudoers.d/vagrant
+echo 'Defaults:vagrant !requiretty' >> /etc/sudoers.d/vagrant
+chmod 0440 /etc/sudoers.d/vagrant
+
+gpasswd -d vagrant wheel
+
+sudo xbps-install -Sy wget
+
+# Installing vagrant keys
+mkdir /home/vagrant/.ssh
+chmod 700 /home/vagrant/.ssh
+cd /home/vagrant/.ssh
+wget --no-check-certificate 'https://raw.github.com/mitchellh/vagrant/master/keys/vagrant.pub' -O authorized_keys
```

Comment:
This might be better off pointing to `https://raw.githubusercontent.com/hashicorp/vagrant/master/keys/vagrant.pub` nowadays. HashiCorp appears to have realized they need to keep things under their name.

Reply:
Good catch.